GSA-TTS/rails-template
# Next gen terraform

# 🎫 Addresses issue: #124 

closes #124 

## 🛠 Summary of changes


* use v2 of the terraform modules
* drop support for `cf push` via manifest.yml and go all-in on terraform for infrastructure & deploy
* re-vamped bootstrap module for ease of use (no more manual copy/paste of output variables [mostly])


## 📜 Testing Plan

How would a peer test this work?

- checkout branch
- run: `export RT_DEV=true`
- run: `exe/rails_template_18f new ../my-rails-app` and answer the questions. use `gsa-tts-devtools-prototyping` as the cg org, and make up a new space name for prod and staging spaces. They should not yet exist.
- move over to the new app and follow the terraform readme to deploy the app to cloud.gov
- ???
- profit

also, `bundle exec rake spec` will run all the tests
